docs/usage.md

```diff
@@ -62,6 +62,48 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Creating a branch
```

Comment:
Reading through this documentation, it seems like we might want two separate sub-comments. One to create a branch and another to merge or base a branch. It seems like the requirements to the two use-cases don't line up well and create an awkward user experience.

For example, why should a user need to provide a revision to create a branch? Shouldn't it just use "HEAD" by default. But for merging and rebase, they should need to provide a revision.

Reply:
Do we want to use 2 separate sub-comments "Creating a branch" and "Updating existing git branches" in the "Handling git branch" section? Maybe I misunderstood the sub-comments, but it seems we use to have 2 sub-comments in one section and you said its kind of confusing. Do we want to change it back to the One large section with 2 sub-comments format?

Reply:
I'm wondering if we should change the code to have 2 separate sub-commands: one for creating a branch and one for merging/rebasing a branch. Since the use-case and options for those sub-commands are different trying to support them in the same sub-command makes things awkward.